suzieq/sqobjects/basicobj.py

```diff
@@ -129,15 +133,30 @@ def _check_input_for_valid_vals(self, good_arg_val_list, **kwargs):
             else:
                 chkval = val
             for v in chkval:
-                if v not in good_arg_val_list[arg]:
+                if v not in good_arg_val_list.get(arg, []):
                     raise ValueError(
                         f"invalid value {val} for argument {arg}")
 
     def validate_get_input(self, **kwargs):
         '''Validate the values of the get function'''
+        fields = self.schema.fields
+        for arg, val in kwargs.items():
+            if arg not in fields or not val or not isinstance(val, list):
+                # only if the value can be a split does a "> 100" become
+                # a two element list instead of being a single element.
+                # This is what the code below is fixing
+                continue
+            if (self.schema.field(arg).get('type', '') in ['int', 'long',
+                                                           'float']):
+                user_val = ' '.join(val)
```

Comment:
While we are here, can't we check whether the syntax of the conditions is valid?

Reply:
Separately. Lets open a ticket on that. It maybe more work than I want to take on in this patch. I want to put out 0.19.0